main/main.go

```diff
@@ -155,3 +164,28 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func restoreIBU(client client.Client) error {
+	ibu := &lcav1alpha1.ImageBasedUpgrade{}
+	filePath := filepath.Join(utils.Host, utils.IBUFilePath)
+	if err := lcautils.ReadYamlOrJSONFile(filePath, ibu); err != nil {
+		if os.IsNotExist(err) {
+			return nil
+		}
+		return err
+	}
+
+	status := ibu.Status
+	if err := client.Create(context.TODO(), ibu); err != nil {
+		if errors.IsAlreadyExists(err) {
+			return nil
+		}
+		return err
+	}
```

Comment:
```suggestion
	if err := client.Create(context.TODO(), ibu); err != nil && !errors.IsAlreadyExists(err) {
		return fmt.Errorf("failed to create ImageBasedUpgrade: %w", err)
	}
```

Reply:
Same as above. Need to skip the status update if there is an existing one.